uberpoet/loccalc.py

```diff
@@ -0,0 +1,67 @@
+#  Copyright (c) 2021 Uber Technologies, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import tempfile
+from os.path import join
+
+from .commandlineutil import count_loc
+from .filegen import SwiftFileGenerator
+from .memoize import memoized
+
```

Comment:
I would suggest putting the static constants for language types in this file.

Reply:
sounds good.

Reply:
I had to put it in another file due to a circular import dependency, however the enum is now there.